### DISCUSSION TITLE
What to fill in url entpoint when add new product

### DISCUSSION THREAD

**Discussion by zhuxiu1234** [Ericsson/codechecker#3481] [Q&A] [answered]
What to fill in url entpoint when add new product

**Comment by csordasmarton** [comment, marked as answer]
@zhuxiu1234 This is a unique identifier for your product which will be available in your URL when you visit a product (e.g.: `java-static-analysis` in https://codechecker-demo.eastus.cloudapp.azure.com/java-static-analysis/runs) on the GUI or you will use it in the command line when you want to store/get data to/from the server.

For more information see: https://github.com/Ericsson/codechecker/blob/master/docs/web/user_guide.md#product_url-format-
